### ISSUE TITLE
RQWorker isn't recognized by Pootle after flush_cache

### ISSUE THREAD

**Issue by JohnnyKing94** [translate/pootle#5741]
Hi guys, i wanted to report you this kind of problem i have with rqworker after running the command `pootle flush_cache --all`. I don't know if it's only me having this problem, but basically that's what happen.
**I link a video below** to make you understand quickly.
I first show in the video that rqworker is running correctly under screen. As you can see, the rqworker is listening and pootle does recognize it in the page.
Then i run `pootle flush_cache --all` and the rqworker (on Pootle end) doesn't seem to be running anymore, but then i show the screen of rqworker and you see the it keeps running.
In conclusion, i terminate the screen and i run again the command `pootle rqworker` (without the screen this time just for showing purposes) and Pootle see the worker running again.

Why this is happening?

LINK VIDEO: https://dl.dropboxusercontent.com/u/89718989/Screenshot/Pootle/rqworker.mp4
